frontend/src/pages/settings/preferences/Preferences.tsx

```diff
@@ -39,10 +46,13 @@ const PreferencePage = () => {
       if (response) {
         dispatch(replaceSettings(response));
       }
+
+      setLoading(false);
     }
 
     retrieveProfile();
-  }, [t, dispatch, contactAdministrator]);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
```

Comment:
I'm not satisfied with this `eslint-disable` here, but I haven't found an elegant solution to remove it yet.

I added it because when I run the `Preferences.spec.tsx` tests with the complete array of dependencies `[t, dispatch, contactAdministrator]`, this effect is triggered two times instead of one, which  makes  the tests fail. When using the app, the effect is only triggered one time as expected.

I first thought it was caused by the fact that the function `dispatch` is "spied", but even with a incomplete dependency array like `[t]` the effect was still triggered two times.